extensions/colormath-ext-jetpack-compose/build.gradle.kts

```diff
@@ -18,6 +18,17 @@ kotlin {
 
     jvm()
 
+    js {
+        browser()
+        nodejs()
+    }
+
+    @Suppress("OPT_IN_USAGE")
```

Comment:
Let's use `@OptIn(ExperimentalWasmDsl::class)` instead. Let's match how we do the declarations for the [main module][1]

[1]: https://github.com/ajalt/colormath/blob/9ff469060467d478466315280c19d803e4dd2bcd/colormath/build.gradle.kts#L14-L17